src/components/keypad.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import Button from './button';
+
+class Keypad extends React.Component {
+  render() {
+    const items = ['AC', '+/-', '%', '/', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
```

Comment:
Currently, when I try to carry out a division operation, an error is displayed which breaks the application, kindly fix this😉.

<img width="1040" alt="image" src="https://github.com/MenendezON/math-magicians/assets/64643995/98bfc38c-907f-483f-95f6-7be09aa849b6">